Compilation Failure (_HASH_SEED) on MSVC 14.15.26726 (Appveyor)

Noticed this while working on my extended mouse PR. It seems that the MSVC version on Appveyor was upgraded from 14.14.26428 (Visual Studio 2017 version 15.7) to 14.15.26726 (Visual Studio 2017 version 15.8) and as a result broke the native Windows compilation:

```
C:\projects\openjdk-jfx\modules\javafx.graphics\src\main\native-glass\win\GlassClipboard.cpp(86): error C2065: '_HASH_SEED': undeclared identifier
```

It appears that `GlassClipboard.cpp` is the only file to reference `_HASH_SEED` (and it uses it once). Not sure why that constant would be un-defined in a newer MSVC version, besides the fact that `hash_map` and `hash_set` were deprecated (at least, the one's from `std` were, maybe the one's from `stdext`, which we use, were not).  Is there a maximum C++ version spec that we can use when developing the native Windows code? Apparently C++11 has `unordered_map` and `unordered_set` equivalents - so that would be a nice long-term solution. A short-term solution may be just defining it ourselves:

`#define _HASH_SEED (size_t)0xdeadbeef`

but of course knowing exactly what changed in the MSVC upgrade to make it so `_HASH_SEED` is not defined would be a prerequisite to a solution.

@johanvos @brcolow, I too see the same issue here and made it build by defining locally. Still working on a WebKit build issue due to multiple definition of a symbol.

--- a/modules/javafx.graphics/src/main/native-glass/win/GlassClipboard.cpp
+++ b/modules/javafx.graphics/src/main/native-glass/win/GlassClipboard.cpp
@@ -29,6 +29,12 @@
 #include <hash_map>
 #include <hash_set>
 
+// The following is derived from _HASH_SEED, which is an internal constant from
+// include/xhash; since this is an internal constant we define our own rather
+// that relying on something that Microsoft could remove (and apparately has in
+// newer version of Visual C++)
+#define GLASS_HASH_SEED (size_t)0xdeadbeef
+
 #include "GlassApplication.h"
 #include "GlassClipboard.h"
 #include "GlassDnD.h"
@@ -83,7 +89,7 @@ size_t hash_value(const FORMATETC &fr)
     _Val += size_t(fr.ptd);
     _Val >>= 13;
     _Val += size_t(fr.tymed);
-    return _Val ^ _HASH_SEED;
+    return _Val ^ GLASS_HASH_SEED;
 }
 
 //NB! There are two suffixes for mimes:

--- a/modules/javafx.graphics/src/main/native-glass/win/GlassClipboard.cpp
+++ b/modules/javafx.graphics/src/main/native-glass/win/GlassClipboard.cpp
@@ -31,8 +31,8 @@
 
 // The following is derived from _HASH_SEED, which is an internal constant from
 // include/xhash; since this is an internal constant we define our own rather
-// that relying on something that Microsoft could remove (and apparately has in
-// newer version of Visual C++)
+// than relying on something that Microsoft could remove (and apparently has in
+// newer versions of Visual C++)
 #define GLASS_HASH_SEED (size_t)0xdeadbeef
 
 #include "GlassApplication.h"

